modules/cborm/models/criterion/Restrictions.cfc

```diff
@@ -169,8 +169,21 @@ component singleton{
 		return restrictions.sizeNE(arguments.property, arguments.propertyValue);
 	}
 	
-	// Use arbitrary SQL to modify the resultset
-	any function sqlRestriction(required string sql){
+	any function sqlRestriction(required string sql, any parameters ){
+        //if we defined a parameter then its assume its a string and pass it through right
```

Comment:
There are many many different types that can be defined so should I make it default to string if not passed in but allow the programmer to define it if they like?